integreat_cms/core/settings.py

```diff
@@ -291,6 +291,17 @@
     os.environ.get("INTEGREAT_CMS_TEXTLAB_API_DEFAULT_BENCHMARK_ID", 420)
 )
 
+#: List of categories for which we want to process HIX feedback details provided by Textlab API
+TEXTLAB_API_FEEDBACK_CATEGORIES: Final[list[str]] = [
+    "moreSentencesInClauses",
+    "moreSentencesInWords",
+    "moreWordsInLetters",
+    "countPassiveVoiceInSentence",
+    "countInfinitiveConstructions",
+    "countNominalStyle",
+    "countFutureTenseInSentence",
+]
```

Comment:
I haven't found anything like this, but if there's a way to generate it from their documentation (maybe swagger), that would be incredibly helpful to prevent api changes from silently breaking our code.

Reply:
Oh my initial comment was a bit misleading: I was thinking about accessing enum types instead of string definitions.

Reply:
Do you mean moving this from settings to a separate enum class?

Reply:
Oops! My initial comment might have been a bit misleading. What I meant was accessing enum types instead of string definitions directly in our code. This way, we can ensure better type safety and catch any api changes early on.

Reply:
I had a quick look at their OpenAPI definition, and it seems that these values are loosely typed in their system either. Please consider this thread solved. 🙈

Reply:
Ah, right, it's hardly possible to parse their docs. We had to contact a supporting person to localize the right fields in their API response 🙈